agent/ecscni/namespace_helper_windows.go

```diff
@@ -74,6 +76,26 @@ func (nsHelper *helper) ConfigureTaskNamespaceRouting(ctx context.Context, confi
 	credentialsAddressRouteAdditionCmd := fmt.Sprintf(ecsBridgeRouteAddCmdFormat, credentialsEndpointRoute, ecsBridgeEndpointName)
 	commands := []string{defaultRouteDeletionCmd, defaultSubnetRouteDeletionCmd, credentialsAddressRouteAdditionCmd}
 
+	if !config.BlockInstanceMetadata {
+		// This naming convention is drawn from the way CNI plugin names the endpoints.
+		taskPrimaryEndpointId := strings.Replace(strings.ToLower(taskENI.MacAddress), ":", "", -1)
```

Comment:
Can you update this comment with a link to the CNI plugin?

Reply:
Sure, I can do that. Since the CNI plugin has not been merged, therefore, we do not have a permanent link for that yet.
Would it do if we add the link in the next final PR?